sources/packages/backend/libs/integrations/src/esdc-integration/disbursement-receipt-integration/disbursement-receipt.processing.service.ts

```diff
@@ -141,6 +150,51 @@ export class DisbursementReceiptProcessingService {
       }
     }
     result.processSummary.push(`Processing file ${remoteFilePath} completed.`);
+    const batchRunDate = responseData.header.batchRunDate;
```

Comment:
Can we create a private method to deal only with the CSV and notification generation?